pkg/controller/common/helpers.go

```diff
@@ -170,6 +170,10 @@ func MergeMachineConfigs(configs []*mcfgv1.MachineConfig, cconfig *mcfgv1.Contro
 		kargs = append(kargs, cfg.Spec.KernelArguments...)
 	}
 
+	// Update kernel arguments based on kernel type for PSI
+	// For realtime kernels, disable PSI; for other kernels, enable it
```

Comment:
I don't think this is valid. There are customers using OpenShift for low latency applications (e.g. Telco) that use the non-RT kernel. If enabling PSI causes a performance degradation then I don't think it is OK to trigger this based on the RT vs. non-RT kernel.

Reply:
I'm happy with the new approach to remove the PSI config from MCO (resulting in it being disabled by default) and then allowing anyone who needs PSI to enable it themselves.

Reply:
I personally do want to investigate a world where we enable by default and have it be easily opted-out of. That needn't be yet, but I would like to be able to rely on the metrics in the platform one day.